dockstore-webservice/src/main/java/io/dockstore/webservice/core/metrics/ValidationStatusCountMetric.java

```diff
@@ -24,52 +24,51 @@
 import io.swagger.v3.oas.annotations.media.Schema;
 import java.util.EnumMap;
 import java.util.Map;
-import javax.persistence.CollectionTable;
-import javax.persistence.ElementCollection;
+import javax.persistence.CascadeType;
 import javax.persistence.Entity;
 import javax.persistence.EnumType;
 import javax.persistence.FetchType;
 import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
 import javax.persistence.MapKeyColumn;
 import javax.persistence.MapKeyEnumerated;
+import javax.persistence.OneToMany;
 import javax.persistence.Table;
+import javax.validation.constraints.NotEmpty;
 import org.hibernate.annotations.BatchSize;
 
 @Entity
 @Table(name = "validation_status")
 @ApiModel(value = "ValidationStatusMetric", description = "Aggregated metrics about workflow validation statuses")
```

Comment:
Was like this before, but since there are two nested classes linked to this now, should Javadoc that this keeps track of the ValidatorTool to ValidatorInfo info map